<launch>
	<arg name="camera_name" default="camera" />
        <node pkg="teleop_vision" type="overlay" name="overlay" output="screen">
        	<param name="cam_intrinsic_calibration_file_path" value="$(find teleop_vision)/resources/flea_left_intrinsic.xml" />
		<param name= "left_image_topic_name" value= "/$(arg camera_name)/left/image_color" />		
		<param name= "right_image_topic_name" value= "/$(arg camera_name)/right/image_color"/>
		<param name= "left_cam_pose_topic_name" value= "/$(arg camera_name)/left/extrinsic_aruco/board_to_camera "/>
		<param name= "frequency" value= "100" /> 
		<param name="image_transport" value="resized"/>
         </node>

</launch>



